kpatch-build/create-diff-object.c

```diff
@@ -2296,7 +2319,10 @@ int main(int argc, char *argv[])
 	struct lookup_table *lookup;
 	struct section *sec, *symtab;
 	struct symbol *sym;
-	char *hint = NULL, *name, *pos;
+	char *hint = NULL, *objname, *pos;
+	char *mod_symvers_path, *pmod_name;
+
+	arguments.args = malloc(sizeof(char *) * NUM_ARGS);
```

Comment:
I think I had changed it to be dynamically allocated because of the shared args.h header, where `struct arguments` is defined. The `args` field was formerly a static array with set size, but since different files had varying number of arguments, I had changed it to `char **` to accommodate differently sized arrays.

But I think you suggested we get rid of args.h anyway (maybe that was overkill), so I'll revert back to static arrays.